src/kfactory/kcell.py

```diff
@@ -1363,6 +1363,16 @@ def get_meta_data(self) -> None:
                     f" Available formats are 'default' or 'legacy'."
                 )
 
+    @property
+    def x(self) -> int:
+        """Returns the x-coordinate of the center of the bounding box."""
+        return self._kdb_cell.bbox().center().x
+
+    @property
+    def y(self) -> int:
+        """Returns the y-coordinate of the center of the bounding box."""
+        return self._kdb_cell.bbox().center().y
+
```

Comment:
do we have center here?

Reply:
i think so, otherwise the test wouldn't pass

Reply:
center was already in KCell.center if that's what you meant

Reply:
Sounds good